### PR TITLE
svg setup, general layout

### PR DESCRIPTION
**Pull request by daimeng**
sorry couldn't figure out how to use figma quickly (i've only ever read others designs). I think its faster if I just make prs.
This has some general setup for the project. I added the svgr plugin to import svg files as react components.

The general layout is pretty typical, two things to note:
- I constrained the body with max-width. I think it might look better light this, especially if we don't have a lot of text content like uniswap.org.
- I think we want the social media on the header of page, especially given we don't have a forum currently.

### CODE REVIEW

**Comment by daimeng** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>f6133f0</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://zodiac-landing-990nlk51k-daimeng.vercel.app'>https://zodiac-landing-990nlk51k-daimeng.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/daimeng/zodiac-landing/8hkxP7Ts177XxSsqJNr9JTXYKKys'>https://vercel.com/daimeng/zodiac-landing/8hkxP7Ts177XxSsqJNr9JTXYKKys</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/daimeng/zodiac-landing/actions/runs/2653402432)

File: components/layout.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import Link from 'next/link'
+import TwitterLogo from './twitter.svg'
+import DiscordLogo from './discord.svg'
+import TelegramLogo from './telegram.svg'
+import Image from 'next/image'
+console.log(TwitterLogo)
```
Comment (kimchicoffee) [inline components/layout.js]: do we want to check in console.log(TwitterLogo)? lol

Comment (daimeng) [reply components/layout.js]: oops, removing

**Comment by daimeng** [comment]
This pull request has been deployed to Vercel.

<table>
<tr>
<td><strong>Latest commit:</strong></td>
<td><code>af74525</code></td>
</tr>
<tr>
<td><strong>✅ Preview:</strong></td>
<td><a href='https://zodiac-landing-d89simsqd-daimeng.vercel.app'>https://zodiac-landing-d89simsqd-daimeng.vercel.app</a></td>
</tr>
<tr>
<td><strong>🔍 Inspect:</strong></td>
<td><a href='https://vercel.com/daimeng/zodiac-landing/7sxLoshGLS4tEAjyRfhTTZEmC7TB'>https://vercel.com/daimeng/zodiac-landing/7sxLoshGLS4tEAjyRfhTTZEmC7TB</a></td>
</tr>
</table>

[View Workflow Logs](https://github.com/daimeng/zodiac-landing/actions/runs/2656307971)

**Pull request merged**